apps/antalmanac/src/components/RightPane/AddedCourses/AddedCoursePane.tsx

```diff
@@ -412,6 +415,7 @@ function AddedSectionsGrid() {
             </Box>
             <Box style={{ marginTop: 50 }}>
                 <Typography variant="h6">{`${scheduleName} (${scheduleUnits} Units)`}</Typography>
+                {courses.length < 1 ? <Typography variant="h7">No Courses Added Yet!</Typography> : <></>}
```

Comment:
```suggestion
                {courses.length < 1 ? <Typography>No Courses Added Yet!</Typography> : <></>}
```
I don't believe `h7` is a valid variant; If you're going for a `<p>` tag, I don't believe the variant prop is necessary.